modules/apps/content-dashboard/content-dashboard-api/bnd.bnd

```diff
@@ -0,0 +1,8 @@
+Bundle-Name: Liferay Content Dashboard API
+Bundle-SymbolicName: com.liferay.content.dashboard.api
+Bundle-Version: 1.0.0
+Export-Package: com.liferay.content.dashboard.item.action
```

Comment:
I don't know if I am seeing two Export-Package lines because of a problem with Github, or we really have two Export-Package lines

Reply:
changing it